build.gradle

```diff
@@ -19,7 +19,7 @@ task wrapper(type: Wrapper) {
 }
 
 buildscript {
-  apply from: "https://github.com/rosjava/rosjava_bootstrap/raw/indigo/buildscript.gradle"
+  apply from: "https://github.com/stratomda/rosjava_bootstrap/raw/indigo/buildscript.gradle"
```

Comment:
any reason for this change? This should probably be reverted before merging, right?